.github/workflows/Linter-Handoff.yml

```diff
@@ -6,7 +6,7 @@ on:
   merge_group:
 jobs:
   handoff:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
Do we need to make this change immediately rather than just waiting for https://github.com/actions/runner-images/issues/10636?

Reply:
We're already using ubuntu-24.04 based images for our armv7l runner, since that requires .NET 9, which isn't supported on older Ubuntu releases.

So this should actually sync up all of our actions to run on similar containers.

Reply:
Fair enough.